docs/dm/zh_CN/maintenance/metrics-alert.md

```diff
@@ -64,11 +50,21 @@ query: `changes(dm_relay_write_error_count{instance="$instance"}[1m])`
 
 告警：需要告警
 
-#### binlog file index
+#### storage remain
 
-query: `dm_relay_binlog_file{instance="$instance"}`
+query: `dm_relay_space{instance="$instance", type="available"}`
```

Comment:
what is the logic for metrics' order? should we need to update the grafana dashboard?

Reply:
It's description docuement, put critical metrics in front, not necessarily a one-to-one correspondence with grafana, but I want to fill these description into grafana dashboard long time, file a issue for it.